bin/options.js

```diff
@@ -1,41 +1,67 @@
 'use strict';
```

Comment:
imo nothing should live in `bin` except the two executables.  maybe we should move this.

Reply:
I agree, will review and change.

Reply:
@boneskull I recommend we move the _options_ module from the `bin` directory in a different change. Does that sound good to you as well?